src/services/github-oauth-service.ts

```
@@ -8,6 +8,7 @@ import {
   createCipheriv,
   createDecipheriv,
 } from "crypto";
+import { Octokit } from "@octokit/rest";
```

ESLint

[error] Unable to resolve path to module '@octokit/rest'.

(import-x/no-unresolved)